src/app/api/session/route.ts

```
@@ -47,128 +107,265 @@ export async function POST(request: Request) {
+          metrics: {
+            totalVolume: 0,
+            totalSets: 0,
+            totalExercises: 0,
+            completedSets: 0,
+            skippedSets: 0,
+            personalRecords: [],
+            volumeRecords: [],
+            adherenceScore: 0,
+          },
+          environment,
+        } as any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)